services/web-app/data/libraries.js

```diff
@@ -289,6 +289,37 @@ const libraryAllowList = {
   }
 }
 
+/**
+ * Libraries to persist to the `.carbon-oss` file system cache so we can use this local data when
+ * deploying to Vercel.
+ */
+const ossLibraries = [
+  'carbon-charts',
+  'carbon-charts-angular',
+  'carbon-charts-react',
+  'carbon-charts-svelte',
+  'carbon-charts-vue',
+  'carbon-styles',
+  'carbon-components',
+  'carbon-angular',
+  'carbon-react',
+  'carbon-svelte',
+  'carbon-vue',
+  'carbon-web-components'
+]
+
+/**
+ * Only use the `ossLibraries` data set if specified in the environment variable.
+ */
+const libraryAllowList = Object.keys(libraries)
+  .filter((key) =>
+    process.env.FILE_SYSTEM_CACHE !== '.carbon' ? ossLibraries.includes(key) : true
+  )
```

Comment:
Similar comment here about using run-mode
```suggestion
  .filter((key) =>
    getRunMode() === DEV ? ossLibraries.includes(key) : true
  )
```